.github/workflows/Build.yml

```
@@ -57,7 +57,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions-rust-lang/setup-rust-toolchain@v1
       - uses: taiki-e/install-action@cargo-llvm-cov
-      - run:  cargo llvm-cov --cobertura --output-path coverage.cobertura.xml
+      - run:  cargo llvm-cov  --all-features --cobertura --output-path  coverage.cobertura.xml
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)